### PR TITLE
Add components repo

### PR DESCRIPTION
**Pull request by yaacov**
Components repo is open for business :tada:

- Add componets repo to `package.json`
- Add start-console script  for starting the console

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,7 +19,8 @@
     "cypress-postreport": "yarn cypress-merge && yarn cypress-generate",
     "test-cypress": "node_modules/.bin/cypress open --config-file ./cypress/cypress.json --env openshift=true",
     "test-cypress-docker": "echo 'visit http://localhost:10000' && DISPLAY=:14 LIBGL_ALWAYS_INDIRECT=0 $(npm bin)/cypress open --config-file ./cypress/cypress.json --env openshift=true",
-    "test-cypress-headless": "node --max-old-space-size=4096 $(npm bin)/cypress run --config-file ./cypress/cypress.json --env openshift=true --headless"
+    "test-cypress-headless": "node --max-old-space-size=4096 $(npm bin)/cypress run --config-file ./cypress/cypress.json --env openshift=true --headless",
+    "start-console": "./start-console.sh"
```
Comment (avivtur) [inline package.json:23]: I can't see `start-console.sh`  where is it coming from?

Comment (yaacov) [reply package.json:23]: Thanks :palm_tree: , added the missing file

File: webpack.config.ts [hunk 2]
```diff
@@ -30,6 +30,8 @@ const config: Configuration = {
   devServer: {
     hot: true,
     port: KUBEVIRT_PLUGIN_PORT,
+    // Allow bridge running in a container to connect to the plugin dev server.
+    allowedHosts: 'all',
```
Comment (yaacov) [inline webpack.config.ts:34]: Added as suggested by @glekner

**Pull request merged**